.github/workflows/build_and_test.yml

```
@@ -339,17 +339,17 @@ jobs:
         run: |
           ${cov_prefix} mold -run cargo build $CARGO_FLAGS $CARGO_FEATURES --bins --tests
 
-      - name: Run cargo test
+      - name: Run nextest test
```

Let's use something tool-independent, for example

```suggestion
      - name: Run Rust tests
```